status/wallet2/account.nim

```diff
@@ -65,6 +69,17 @@ proc toOpenseaCollection*(jsonCollection: JsonNode): OpenseaCollection =
         ownedAssetCount: jsonCollection{"owned_asset_count"}.getInt
     )
 
+proc getOpenseaTraits*(jsonAsset: JsonNode): seq[OpenseaTrait] =
+    var traitList: seq[OpenseaTrait] = @[]
+    for index in jsonAsset{"traits"}.items:
+        var newTrait: OpenseaTrait
+        new newTrait
+        newTrait.traitType = index["trait_type"].getStr
+        newTrait.value = index["value"].getStr
+        newTrait.displayType = index["display_type"].getStr
```

Comment:
nit: you can construct the trait directly:
```
OpenseaTrait(traitType: index{"trait_type"}.getStr, value: index{"value"}.getStr, displayType: index{"display_type"}.getStr)
```

Also see the usage of `{` rather than `[` which will handle nil values